interface/views.py

```diff
@@ -307,3 +307,23 @@ def user_page(request, username):
     return render(request, 'interface/user_page.html', {
         'submissions': submissions,
     })
+
+
+@login_required
+@staff_member_required
+def reveal(request, course_pk, assignment_pk):
+    course = get_object_or_404(Course, pk=course_pk)
+    assignment = get_object_or_404(course.assignment_set, pk=assignment_pk)
+
+    if (request.user
+            not in course.teaching_assistants.all()):
```

Comment:
It might look better if the condition is extracted into a variable. Something like:
```
is_ta = request.user in course.teaching_assistants.all()
if not is_ta:
   #bad day
```